rad/monadic/names-remote.rad

```diff
@@ -0,0 +1,75 @@
+(import prelude/basic :unqualified)
+(import prelude/bool :unqualified)
+(import prelude/dict :unqualified)
+(import prelude/ref :unqualified)
+(import prelude/validation :as 'validator)
+(import prelude/patterns :unqualified)
+(import prelude/util :unqualified)
+(import prelude/lens :unqualified)
+
+(def names (ref []))
```

Comment:
any reason for not using `{}`?

Reply:
It feels like we want these to be look-upable and unique both ways, and between having two dicts or one list of pairs, it seemed like a toss.

Reply:
:+1: